io.edgehog.devicemanager.OSImage.json

```diff
@@ -0,0 +1,29 @@
+{
+  "interface_name": "io.edgehog.devicemanager.OSImage",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "device",
+  "mappings": [
+    {
+      "endpoint": "/branch",
+      "type": "string",
+      "description": "Source branch that generates the image."
+    },
+    {
+      "endpoint": "/hash",
+      "type": "string",
+      "description": "SHA256 of the image"
+    },
+    {
+      "endpoint": "/name",
+      "type": "string",
+      "description": "Name of the image"
+    },
+    {
+      "endpoint": "/version",
+      "type": "string",
+      "description": "Version of the image"
+    }
```

Comment:
I think we might add a `/buildId` that should be a human readable build id, such as date+time string or date-time-git-commit or whatever make easier identifying how it has been built.

Reply:
++ this

Reply:
I'm not sure how much "human-readable" could such info be against a "simple" version number, but if we want to keep both I don't have anything against it.